### PR TITLE
fix: SV indicator type increase font size and spacing DHIS2-7420

### PR DESCRIPTION
**Pull request by edoardo**
Implements [DHIS2-7420](https://jira.dhis2.org/browse/DHIS2-7420)

---

### Key features

1. Increase font size for the indicator type text under the single value
2. Increse the spacing between the valuer and the indicator type text

### Screenshots

Before:
Screenshot 2021-09-07 at 16 00 22

After:
Screenshot 2021-09-07 at 16 04 21

### CODE REVIEW

File: src/visualizations/config/generators/dhis/singleValue.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ const generateDashboardItem = (config, legendSet) => {
             config.value,
             config.formattedValue,
             config.subText,
-            legendSet
+            legendSet,
+            40
```
Comment (jenniferarnesen) [inline src/visualizations/config/generators/dhis/singleValue.js:110]: maybe use a constant throughout the file?

Comment (edoardo) [reply src/visualizations/config/generators/dhis/singleValue.js:110]: There are 2 different values, do you mean to have 2 constants each used in one place?

Comment (jenniferarnesen) [reply src/visualizations/config/generators/dhis/singleValue.js:110]: So it's just coincidence that both use the number 40?

**Review by jenniferarnesen — APPROVED** [review]

File: src/visualizations/config/generators/dhis/singleValue.js (hunk 1 above)
Comment (edoardo) [reply src/visualizations/config/generators/dhis/singleValue.js:110]: Yes, one is for the font size, the other is for the `y` attribute on the SVG text node to control the spacing with the value above.

**Pull request merged**